clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -160,6 +169,16 @@ export async function activate(
     ),
   );
 
+  context.subscriptions.push(
+    vscode.window.onDidChangeActiveTextEditor(() => {
```

Comment:
We don't really care that much about the active editor being changed, we should listen to `onDidChangeTextDocument` and possibly `onDidCloseTextDocument` to monitor document content changes.

Reply:
Fixed